### PR TITLE
Document that deploy API accepts email

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -456,6 +456,7 @@ Key | Example
 --- | -------
 environment | production
 username | john
+email | john@smith.com
```
Comment (kyrylo) [inline source/index.html.md]: We should probably add this info to line 444, too.

**Comment by vmihailenco** [comment]
Done, ptal

**Comment by kyrylo** [comment]
What about line 444?

> ```shell
 curl -X POST -H "Content-Type: application/json" -d '{"environment":"production","username":"john","repository":"https://github.com/airbrake/airbrake","revision":"38748467ea579e7ae64f7815452307c9d05e05c5","version":"v2.0"}' "https://airbrake.io/api/v4/projects/PROJECT_ID/deploys?key=PROJECT_KEY"
 ```

**Comment by vmihailenco** [comment]
Sorry, done. PTAL

File: source/index.html.md [hunk 2]
```diff
@@ -506,6 +509,8 @@ curl "https://airbrake.io/api/v4/projects/PROJECT_ID/deploys/DEPLOY_ID?key=USER_
   "deploy": {
     "environment": "production",
     "username": "john",
+    "username": "john",
```
Comment (shifi) [inline source/index.html.md]: I'm too late. Now we have 2 Johns.

Comment (vmihailenco) [reply source/index.html.md]: thanks, quick fixed in master

**Pull request merged**